### PR TITLE
Pin versions to enable repeatable build

### PR DESCRIPTION
**Pull request by janderson2**
It was not possible to rebuild a working version of the app due to issues with breaking changes in transient dependencies. This change pins the versions to match the build currently deployed to production. There are a number of upgrades required, but these will need to be completed separately. The versions are pinned by using fixed versions in the package.json, introducing a package-lock.json generated from the live docker image and pinning the node version for the docker image.

❕ This PR adds an `npm-shrinkwrap.json` file to achieve the repeatable build. This isn't best practice, but it pins all transient dependencies until we can complete a general upgrade on the app.

In addition to fixing the build, this PR also:

* Adds an .nvmrc to specify the node version for local development
* Adds a Makefile with our standard targets to make this repo more accessible
* Adds a LICENCE file
* Updates the README inline with the changes and to fix the invalid POST command for testing whether the API is functional

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,57 @@
...
-$ curl http://localhost:8080
-```
+To run the API locally you must have the following:
 
-To post content to the service for conversion, put your content in the "input" field using a application/x-www-form-urlencoded format:
+1. [nvm](https://github.com/nvm-sh/nvm) installed:
 
+   ```shell
+   brew install nvm
+   ```
```
Comment (lindenmckenzie) [inline README.md:17]: Didn't Nick have a word with you about this? Installing NVM via brew has issues with updating.

File: README.md [hunk 2]
```diff
@@ -1,27 +1,57 @@
...
+   nvm install
+   ```
+
+### Build and run
+
+Once you have node installed, run the following to install the dependencies and run the server:
+
+```shell
+make debug
 ```
```
Comment (lindenmckenzie) [inline README.md:45]: indentation.

**Review by lindenmckenzie — CHANGES_REQUESTED** [review]
```
SyntaxError: Unexpected token ...
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/lindenmckenzie/git/mathjax-api/node_modules/mathjax-node/node_modules/jsdom/node_modules/tough-cookie/lib/pubsuffix-psl.js:32:11)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
make: *** [debug] Error 1
```

On `make debug`

File: README.md (hunk 2 above)
Comment (janderson2) [reply README.md:45]: This isn't under a numbered list like the others so I believe the indentation is correct.

File: README.md (hunk 1 above)
Comment (janderson2) [reply README.md:17]: Will message you about this as Nick didn't mention any updating issues.

**Review by lindenmckenzie — APPROVED** [review]

**Pull request merged**